modules/scripts/config.py

```diff
@@ -101,6 +102,7 @@ def _get_env_dict(conf):
     build = conf['build']
     publish = conf['publish']
     extras = conf['extras']
+    social = conf['social']
```

Comment:
It was not what i wanted. It was what i interpreted from the issue description and expected you to discuss. 
Anyway, if it is not making sense, remove it